### PR TITLE
Basic Koans about structs

### PR DESCRIPTION
**Pull request by felipesere**

### CODE REVIEW

File: lib/koans/08_structs.ex [hunk 1]
```diff
@@ -0,0 +1,59 @@
+defmodule Structs do
+  use Koans
+
+  defmodule Person do
+    defstruct [:name, :age ]
+  end
+
+  koan "structs are defined and named after a module" do
+    assert %Person{}
```
Comment (ukutaht) [inline lib/koans/08_structs.ex]: I understand the intention here but where would we leave a blank here? `assert __`  feels a bit strange to me

File: lib/watcher.ex [hunk 2]
```diff
@@ -4,7 +4,7 @@ defmodule Watcher do
   def callback(file, events)  do
     if Enum.member?(events, :modified) do
       try do
-        [{mod, _}] = Code.load_file(file)
+        [{mod, _} | _] = Code.load_file(file)
```
Comment (ukutaht) [inline lib/watcher.ex]: What does this change?

Comment (felipesere) [inline lib/watcher.ex]: Structs are defined within a module, when I add a 
```
defmodule MyStruct do
  defstruct ...
defmodule
``` 
within a koan I get more than a single module. 
The previous version forced a single element in the list and failed the match.

File: lib/koans/08_structs.ex (hunk 1 above)
Comment (felipesere) [inline lib/koans/08_structs.ex]: Yeah, I wanted some way to show how to create structs without going to deep. I can remove that koan.

**Pull request merged**